### PR TITLE
Employee page

### PR DESCRIPTION
**Pull request by mikkelcodes**
I plan to refactor some components further, especially the tagbar component is a bit complex now and has room for improvement.

### CODE REVIEW

File: packages/website/src/hooks/useWindowDimensions.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+// import { window } from 'browser-monads';
```
Comment (mikkelcodes) [inline packages/website/src/hooks/useWindowDimensions.js:2]: If deployment to azure gives error due to window == undefined, try uncommenting this code.

Comment (TheKnarf) [reply packages/website/src/hooks/useWindowDimensions.js:2]: It does indeed look like you need to uncomment this to get the build working.

File: packages/website/src/layout.css [hunk 2]
```diff
@@ -12,3 +12,11 @@
     }
   }
 }
+
+iframe .ytp-large-play-button,
+iframe .ytp-button {
+  display: none;
+}
+#movie_player > div.ytp-cued-thumbnail-overlay > button {
```
Comment (TheKnarf) [inline packages/website/src/layout.css:20]: What is this used for?

Comment (mikkelcodes) [reply packages/website/src/layout.css:20]: fixed

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**